C++/C.sublime-syntax

```diff
@@ -614,6 +669,9 @@ contexts:
   eol-pop:
     - match: $\n?
       pop: 1
+  consume-pop:
```

Comment:
I'm very suspicious of this context. Are you using it somewhere?

Reply:
oh i guess i removed whatever functionality it had, so i will remove it in the next commit